pwa/app/routes/event.$eventKey.tsx

```diff
@@ -375,11 +385,20 @@ function TeamsTab({
                   {t.city}, {t.state_prov}, {t.country}
                 </div>
               </div>
-              <img
-                src="https://placehold.co/400x400"
-                alt=""
-                className="size-20"
-              />
+              {(() => {
+                const maybeImage = getTeamRobotPic(
+                  media.filter((m) => m.team_keys.includes(t.key)),
+                );
+
+                return maybeImage === undefined ? null : (
+                  <img
```

Comment:
On prod this is a carousel. Do we want to replicate that?

Reply:
IMO we don't need to. I don't think it adds a ton of value. We could always add the carousel to the team-event modal.